cfme/tests/containers/test_ssl_providers.py

```diff
@@ -59,12 +60,20 @@ def test_add_provider_naming_conventions(provider, soft_assert):
             - Integer name
         * Assert that provider was added successfully with each of those
     """
+    token = provider.process_credential_yaml_key(
+        provider.get_yaml_data()['credentials'],
+        cred_type='token'
+    ).token
     for provider_name in provider_names:
-        prov = OpenshiftProvider(
+        default_endpoint = OpenshiftDefaultEndpoint(
```

Comment:
This is just comment not remark. you can remove/replace args which aren't needed or should have another value.